pkg/virt-operator/creation/components/deployments.go

```diff
@@ -298,7 +332,7 @@ func NewApiServerDeployment(namespace string, repository string, version string,
 		{
 			Name:          "metrics",
 			Protocol:      corev1.ProtocolTCP,
-			ContainerPort: 8443,
+			ContainerPort: 9443,
```

Comment:
why was this port changed?

Reply:
Changed back. Now everything is served on one port.